patient/models.py

```diff
@@ -26,13 +27,17 @@ class Patient(models.Model):
     last_name = models.CharField(max_length=50)
     first_name_pinyin = models.CharField(max_length=50)
     last_name_pinyin = models.CharField(max_length=50)
-    gender = models.CharField(max_length=1, choices=(
-        ('M', 'Male'),
-        ('F', 'Female'),
-        ('O', 'Other')
-    ))
+    gender = models.IntegerField(choices=(
+        (0, 'M'),
+        (1, 'F'),
+        (2, 'O')
+    ),default=0)
     birthdate = models.DateField()
-    relationship = models.CharField(max_length=50)
+    relationship = models.IntegerField(choices=(
+        (0, 'SELF'),
```

Comment:
Should we just do integer here? Keep a very loose restriction and open for mapping extension

Reply:
see new commit for update